python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -0,0 +1,63 @@
+import collections
+import re
+from lsst.pipe.tasks.ingestCalibs import CalibsParseTask
+
+class DecamCalibsParseTask(CalibsParseTask):
+    def getInfo(self, filename):
+        """Get information about the image from the filename and/or its contents
+
+        @param filename: Name of file to inspect
+        @return File properties; list of file properties for each extension
+        """
+        phuInfo, infoList = CalibsParseTask.getInfo(self, filename)
+        # Single-extension fits without EXTNAME can be a valid CP calibration product
+        # Use info of primary header unit
+        if not infoList:
+            infoList.append(phuInfo)
+        return phuInfo, infoList
+
+    def translate_ccdnum(self, md):
+        """Return CCDNUM as a integer
+
+        @param md (PropertySet) FITS header metadata
+        """
+        if md.exists("CCDNUM"):
+            ccdnum = md.get("CCDNUM")
+        else:
+            self.log.warn("Unable to find value for CCDNUM")
+            ccdnum = None
+        # Some MasterCal from NOAO Archive has 2 CCDNUM keys in each HDU
+        # Make sure only one integer is returned.
+        if isinstance(ccdnum, collections.Sequence):
+            try:
+                ccdnum = ccdnum[0]
+            except IndexError:
+                ccdnum = None
+        return ccdnum
+
+    def translate_date(self, md):
+        """Extract the date or date-time string in ISO 8601 format
+        from the supplied FITS header and return a mere date.
+        Return "unknown" if the DATE-OBS header can not be located.
```

Comment:
Please fix the docs to also indicate that "unknown" will be returned if the header is there but is corrupt.